contentcuration/contentcuration/views.py

```diff
@@ -14,7 +14,7 @@
 from kolibri.content.models import File
 
 def base(request):
-    return redirect(channel_list)    # redirect to the channel list page
+    return redirect('channels')    # redirect to the channel list page
```

Comment:
It uses the url pattern name, since reverse is automatically called on the argument. It then resolves it to the `channel_list` view.